packages/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -39,7 +39,7 @@
     "@typescript-eslint/parser": "5.8.1",
     "codecov": "3.8.3",
     "colors": "1.4.0",
-    "eslint": "7.32.0",
+    "eslint": "8.7.0",
```

Comment:
not related to the version bump, but should we remove `eslint` dependency and other eslint related packages from the subpackages, and keep it only in root?
```json
    "@typescript-eslint/eslint-plugin": "5.8.1",
    "@typescript-eslint/parser": "5.8.1",
    "eslint": "7.32.0",
    "eslint-config-airbnb-base": "14.2.1",
    "eslint-plugin-header": "3.1.1",
    "eslint-plugin-import": "2.23.4",
    "eslint-plugin-json5": "0.1.3",
```

Reply:
Not sure we can since the deps aren't hoisted, this would mean that you couldn't run eslint independently for a specific repo ?

Reply:
This dependency is only specified on `browser-extension-autoinject` and `instrumentation-aws-sdk`.
If it's needed per package then we should also add it to all other packages in the repo, but I think it's better to configure and run lint on the entire repo from the root, and not per package.

Reply:
`browser-extension-autoinject` use `eslint-plugin-json5`. 
I believe it can be hoisted to the repo root.